google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -892,7 +892,7 @@ public static BlobListOption fields(BlobField... fields) {
     private final Object value;
 
     enum Option {
-      HTTP_METHOD, CONTENT_TYPE, MD5, EXT_HEADERS, SERVICE_ACCOUNT_CRED
+      HTTP_METHOD, CONTENT_TYPE, MD5, EXT_HEADERS, SERVICE_ACCOUNT_CRED,SERVICE_ENDPOINT
```

Comment:
```suggestion
      HTTP_METHOD, CONTENT_TYPE, MD5, EXT_HEADERS, SERVICE_ACCOUNT_CRED, SERVICE_ENDPOINT
```

Reply:
Let's keep the Enum `SignUrlOption.Option.SERVICE_ENDPOINT` and `SignUrlOption.withHostName()` consistent. Change `SERVICE_ENDPOINT` to `HOST_NAME`.